client/ayon_resolve/plugins/create/create_editorial_package.py

```diff
@@ -1,88 +1,120 @@
 import json
 from copy import deepcopy
-from ayon_core.tools.context_dialog.window import (
-    ContextDialog,
-    ContextDialogController
-)
-from ayon_core.pipeline import get_current_project_name
-from ayon_api import get_folder_by_id, get_task_by_id, get_folder_by_path
-from ayon_core.pipeline.create.legacy_create import LegacyCreator
 
-from ayon_resolve.api import lib
-from ayon_resolve.api.plugin import get_editorial_publish_data
+from ayon_core.pipeline.create import CreatorError, CreatedInstance
 
+from ayon_resolve.api import lib, constants
+from ayon_resolve.api.plugin import ResolveCreator, get_editorial_publish_data
 
-class CreateEditorialPackage(LegacyCreator):
+
+class CreateEditorialPackage(ResolveCreator):
     """Create Editorial Package."""
 
-    name = "editorial_pkg"
+    identifier = "io.ayon.creators.resolve.editorial_pkg"
+    product_name = "editorial_pkgMain"
     label = "Editorial Package"
     product_type = "editorial_pkg"
     icon = "camera"
     defaults = ["Main"]
 
-    def process(self):
-        """Process the creation of the editorial package."""
-        project_name = get_current_project_name()
-        folder_path = self.data["folderPath"]
-
-        current_folder = get_folder_by_path(project_name, folder_path)
-        print(current_folder)
+    def create(self, subset_name, instance_data, pre_create_data):
+        """
+        """
+        super(CreateEditorialPackage, self).create(subset_name,
+                                           instance_data,
+                                           pre_create_data)
```

Comment:
Should be fine like this?
```suggestion
        super().create(subset_name,
                       instance_data,
                       pre_create_data)
```